core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2117,4 +2117,12 @@ package object config {
       // batch of block will be loaded in memory with memory mapping, which has higher overhead
       // with small MB sized chunk of data.
       .createWithDefaultString("3m")
+
+  private[spark] val MARK_FILE_LOST_ON_EXECUTOR_LOST =
+    ConfigBuilder("spark.shuffle.markFileLostOnExecutorLost")
+      .doc("Mark shuffle file lost when executor is lost. People could set this to false when " +
+       "using remote shuffle services because the shuffle file is not stored on the executor.")
```

Comment:
nit: What about if we reworded this to something like:
"Mark shuffle files as lost when an executor is lost. If you are using a remote shuffle service such that shuffle files are not stored on the executor consider setting this to false."

Reply:
This is good suggestion, will do the rewording, thanks!

Reply:
@holdenk I updated the wording just now.